modules/openapi-generator/src/main/resources/kotlin-server/libraries/ktor/api.mustache

```diff
@@ -23,7 +22,6 @@ import {{packageName}}.infrastructure.ApiPrincipal
 
 {{#operations}}
 fun Route.{{classname}}() {
-    val gson = Gson()
```

Comment:
looks like a breaking change to me.

can we keep it somehow as part of the upgrade?

Reply:
Can you please explain what kind of compatibility must be preserved?

Reply:
can you please PM me via Slack when you've time? 

https://join.slack.com/t/openapi-generator/shared_invite/zt-2uoef5v0g-XGwo8~2oJ3EoziDSO1CmdQ